AdminWebsite/AdminWebsite.UnitTests/Extensions/DateExtensionTests.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using AdminWebsite.Extensions;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace AdminWebsite.UnitTests.Extensions;
+
+public class DateExtensionTests
+{
+    
+    [TestCase("2023, 01, 05", 1)]
+    [TestCase("2022, 12, 30", 52)]
+    public void Should_get_correct_week_of_the_year(DateTime testDate, int validWeek) 
```

Comment:
what happens when an invalid date is given?

Reply:
the method being tested takes a datetime object, it can't receive an invalid date by it's nature. If you try to pass the 31st of feb into the constructor of the datetime class it will throw an exception.

Reply:
<img width="676" alt="image" src="https://user-images.githubusercontent.com/38870219/213400847-43b8b1a1-da62-4854-a12d-bf719e9a6fb9.png">